android/app/build.gradle

```diff
@@ -22,16 +22,19 @@ if (flutterVersionName == null) {
 }
 
 apply plugin: 'com.android.application'
+// START: FlutterFire Configuration
+apply plugin: 'com.google.gms.google-services'
+// END: FlutterFire Configuration
```

Comment:
do you need it information?))

Reply:
not mine ) autogenerated after 
$flutterfire config

Reply:
remove the comments plz